src/device-registry/utils/create-activity.js

```diff
@@ -265,10 +265,10 @@ const createActivity = {
           deviceBody["body"] = {};
           deviceBody["query"] = {};
           deviceBody["body"]["height"] = height;
-          deviceBody["body"][" mountType"] = mountType;
-          deviceBody["body"][" powerType"] = powerType;
-          deviceBody["body"][" isPrimaryInLocation"] = isPrimaryInLocation;
```

Comment:
just removes the white space from the properties of the objects before being used for deployment purposes.